### PR TITLE
Using Bootstrap / Newsweek Clone page

### PR DESCRIPTION
**Pull request by GabrielJSuarez**
- Created clone of the Newsweek main page using HTML5 / CSS 3 / Bootstrap 4
- Created index.html with semantic code
- used bootstrap boilerplate to get all the functionality
- Created an external style sheet for precise styling
- No local images used
- Some Jquery functionality was required to accomplish the desired mobile layout but it was recreated cloning HTML code and using CSS to display it when necessary.

### CODE REVIEW

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

 WOW, what project! ⭐


#### Hey @GabrielJSuarez, Is a pleasure to review such a good project! Congrats for the amazing work you already did, the project looks awesome! It breaks my heart 💔 not approve it right away, but I swear, we're almost done, there are just some minor changes that you guys have to implement to proceed:

  _Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.
  
## [REQUIRED CHANGES]
#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
- [ ] Please implement the fixed navbar in small, medium, and large screens.
- [ ] That logo is way too big, please fix that 
#### Large screen issue:
image

#### Good practices REQUIREMENTS (Those changes to make you code like a pro 👽)⤵
- [ ] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image

- [ ] You're overusing (13x) the `!important` rule at your css code, please remove them.
Tip: you can use !important, but you shouldn't overuse it.

## [OPTIONAL CHANGES]
- [ ] Add a favicon to make it more similar to the original page;
- [ ] Put css files inside a css folder and then put that css folder inside that assets file that you already have.
- [ ] Put img files inside a img folder.

## Summary
✨ Your page looks amazing! 👏
✨ Well done on that descriptive commits! 🔝

## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

  👨🏻‍💻 **TSE: Kevin Alves**

    

  📫 Have any question? Better call Kevin:

  <a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

    
  <sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Hello again @GabrielJSuarez, thanks for improving some of my required changes and my optional changes also! That's awesome 🚀! However... Looks like you forgot to implement one of my requirements 😕 (CSS repeated code and !important overuse)

  _Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.
  
## [REQUIRED CHANGES]

#### Good practices REQUIREMENTS (Those changes to make you code like a pro 👽)⤵
- [ ] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image

- [ ] You're still overusing (10x) the `!important` rule at your css code, please remove them.
Tip: you can use !important, but you shouldn't overuse it.


## Summary
✨ Your page looks even more amazing now! 👏🥇
✨ Well done on making the previously requested changes! 💯
✨ Thanks for implementing those optional changes also ⭐

That one was close! 😅


## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

  👨🏻‍💻 **TSE: Kevin Alves**

    

  📫 Have any question? Better call Kevin:

  <a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

    
  <sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

File: assetts/css/style.css [hunk 1]
```diff
@@ -0,0 +1,595 @@
...
+
+.bg-orange {
+  color: #fff;
+  padding: 0 0.5rem;
+}
+
+@media (min-width: 768px) {
+  .no-border {
+    border: none !important;
+  }
```
Comment (kevcap) [inline assetts/css/style.css]: There's no need to wrap it in a media query

File: assetts/css/style.css [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+
+/* OPINION SECTION */
+
+.opinion-title {
+  border-bottom: none;
+}
+
+.opinion-section:last-child {
+  border: none;
+}
```
Comment (kevcap) [inline assetts/css/style.css]: Keep your code DRY. This same rule was already used alone.

File: assetts/css/style.css [hunk 3]
```diff
@@ -0,0 +1,595 @@
...
+
+.subscribe-input::placeholder {
+  color: #555;
+  text-indent: 9px;
+  font-weight: 500;
+}
+
+.sing-up-btn {
+  border: none;
+}
```
Comment (kevcap) [inline assetts/css/style.css:280]: Keep your code DRY. This same rule was already used alone.

File: assetts/css/style.css [hunk 4]
```diff
@@ -0,0 +1,595 @@
...
+
+@media (min-width: 768px) {
+  .copyright {
+    font-size: 1.6rem;
+    margin-right: auto;
+  }
+
+  .social-media {
+    border: none !important;
+  }
```
Comment (kevcap) [inline assetts/css/style.css:579]: Keep your code DRY. This same rule was already used alone.

File: assetts/css/style.css [hunk 5]
```diff
@@ -0,0 +1,592 @@
...
+  font-size: 1.3rem;
+  line-height: 18px;
+}
+
+/*------------------------------*/
+
+/* DEBATE SECTION */
+
+.border-thick {
+  border-width: 5px !important;
```
Comment (kevcap) [inline assetts/css/style.css]: Please, remove that `!important` rule 🗑

File: assetts/css/style.css [hunk 6]
```diff
@@ -0,0 +1,592 @@
...
+
+/* FOOTER */
+
+.footer-logo {
+  width: 236px;
+  fill: #f72210;
+}
+
+.change-editions {
+  color: #666 !important;
```
Comment (kevcap) [inline assetts/css/style.css]: Please, remove that `!important` rule 🗑

File: assetts/css/style.css [hunk 7]
```diff
@@ -0,0 +1,592 @@
...
+
+/* MEDIA QUERIES */
+
+/* TABLET */
+
+@media (min-width: 768px) {
+  /* GLOBALS */
+
+  .no-border {
+    border: none !important;
```
Comment (kevcap) [inline assetts/css/style.css]: Please, remove that `!important` rule 🗑

File: assetts/css/style.css [hunk 8]
```diff
@@ -0,0 +1,592 @@
...
+  /* DEBATE SECTION */
+
+  .debate-img {
+    width: 10%;
+  }
+
+  /* NEWSLETTER SECTION */
+
+  .newsletter-container {
+    width: 40% !important;
```
Comment (kevcap) [inline assetts/css/style.css]: Please, remove that `!important` rule 🗑

File: assetts/css/style.css [hunk 9]
```diff
@@ -0,0 +1,592 @@
...
+  /* SUBSCRIPTION CONTAINER */
+
+  .subscription-text {
+    padding: 4rem 0;
+  }
+
+  .subscription-text h5,
+  .subscription-text p {
+    font-size: 2.2rem;
+    text-align: left !important;
```
Comment (kevcap) [inline assetts/css/style.css]: Please, remove that `!important` rule 🗑

File: assetts/css/style.css [hunk 10]
```diff
@@ -0,0 +1,592 @@
...
+  }
+
+  /*NEWSLETTER */
+
+  .newsletter-text h3 {
+    font-size: 2.5rem;
+  }
+
+  .newsletter-text p {
+    text-align: left !important;
```
Comment (kevcap) [inline assetts/css/style.css]: Please, remove that `!important` rule 🗑

File: assetts/css/style.css [hunk 11]
```diff
@@ -0,0 +1,592 @@
...
+  .copyright {
+    font-size: 1.6rem;
+    margin-right: auto;
+  }
+
+  .social-media a {
+    font-size: 2.5rem;
+  }
+  .change-editions {
+    border-right: 1px solid #dee2e6 !important;
```
Comment (kevcap) [inline assetts/css/style.css]: Please, remove that `!important` rule 🗑

File: assetts/css/style.css [hunk 12]
```diff
@@ -0,0 +1,592 @@
...
+  }
+
+  .debate-title {
+    font-size: 1.7rem;
+  }
+
+  /* NEWSLETTER SECTION */
+
+  .newsletter-container {
+    width: 80% !important;
```
Comment (kevcap) [inline assetts/css/style.css]: Please, remove that `!important` rule 🗑

File: assetts/css/style.css [hunk 13]
```diff
@@ -0,0 +1,592 @@
...
+
+  .img-front {
+    left: 30%;
+  }
+
+  /*NEWSLETTER */
+
+  .newsletter-text h3 {
+    font-size: 2.4rem;
+    width: 100% !important;
```
Comment (kevcap) [inline assetts/css/style.css]: Please, remove that `!important` rule 🗑

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Hey @GabrielJSuarez, good job implementing the previously required changes, your project is looking good, however, to accomplish as the actual page, you required to do these changes, listed below:
 

## [REQUIRED CHANGES]
- [ ] As I previously requested, you have to remove all those `!important` rules from your CSS code.
image
I added some comments down below to help you 💪
(If you're having issues with that request, we can have **another** video call so we can figure out what's going on, for the first time we was able to solve that logo issue! 😎🔧)

- [ ] Please, in order to keep your repo professional, check the description of your commits, they should be descriptive so the other programmer can see exactly what you changed with that commit. [Check this link](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
image
([Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/))

#### You were doing great with those commits before, but that last commit doesn't look descriptive ⤵
image


## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by kubilaycaglayan — CHANGES_REQUESTED** [review]
Hello Team :wave: 
Status: Changes Required :recycle: 

Great job so far! You have done all the required changes!
There's one important thing before we move on :arrow_down: 

## GitHub Flow

- [ ]  Set your `master` branch as your default branch. Currently your default branch is `newsweek-Clone-Feature-Branch`. But in Microverse we are working with the `master` default branch.  :pray: 

:bulb: Probably this is a mistake you did involuntarily. Please don't forget, this might make your project invalid next time. You will learn more about default branches and different branching methodologies in the next chapters of the Microverse.
#### :question: Why it happened? 
:point_right: If `newsweek-Clone-Feature-Branch` was the first branch you have created then GitHub automatically makes it default.
#### :question: How can you fix it?
:point_right: You can easily change it from repository settings. 

### That's it. After doing the changes, you can request another review.

With all my respect... :pray:

**Comment by GabrielJSuarez** [comment]
Are you for real?

**Comment by kevincappuccino** [comment]
### Thanks @kubilaycaglayan for noting that issue. 
#### Quick advice as a Microverse student and a Developer
@GabrielJSuarez ➡ Microverse is a school. That school has <strong>standards</strong> and it means that you <strong>must follow its standards</strong> in order to get your project approved. It will happen a lot in your professional life, because industries and companies like Google, Facebook, even local businesses and <strong>single clients</strong> also have tons of <strong>standards, rules, and cultures</strong>, and if you want to be part of any of them, you have to get used to it and deal with it in the most professional way always.

If you need anything, you can reach me through slack, or for technical questions/issues, you also can contact a Fulltime TSE.

With you the very best and see you at the top 🚀

Warm regards!

**Comment by GabrielJSuarez** [comment]
The rule is to not work on the master branch, there's nothing about what
branch is the default branch but ok.

**Review by kubilaycaglayan — APPROVED** [review]
Hello Team :wave: 
Status: Project Approved :green_circle: 

You are great! :blush: 

With all my respect...

**Pull request merged**